### PR TITLE
Added a configurable maintenance page (similar to the one on identity

### PR DESCRIPTION
**Pull request by michalpiatek**
service)

### CODE REVIEW

File: src/ui/page/maintenance.html [hunk 1]
```diff
@@ -0,0 +1,1179 @@
...
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-22141655-4">
+    </script>
+    <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag() { dataLayer.push(arguments); }
+        gtag('js', new Date());
+        gtag('config', 'UA-22141655-4', { anonymize_ip: true });
+    </script>
+    <style>
```
Comment (charlesluokb) [inline src/ui/page/maintenance.html]: can we move the style configurations (i.e. scss stuff) to a separate file to make this page more readable?

File: src/ui/server.ts [hunk 2]
```diff
@@ -20,12 +20,14 @@ import * as i18n from 'lib/service/translation'
 import {ProfileChecker} from 'lib/ui/profileChecker'
 import * as template from 'lib/ui/template'
 
+import * as maintenanceMiddleware from 'lib/service/maintenance'
```
Comment (charlesluokb) [inline src/ui/server.ts]: as per naming convention, I think people are more familiar with `xxxxService` :-) can we call it `maintenanceService` ? be in line with its parent folder name i.e. `lib/service`.

**Comment by charlesluokb** [comment]
I have not seen spec tests (under /webdriver/src/test) for the maintenance page. are they in different location? can we have them written? @michalpiatek

File: src/ui/server.ts (hunk 2 above)
Comment (michalpiatek) [reply src/ui/server.ts]: sure

File: src/ui/page/maintenance.html (hunk 1 above)
Comment (michalpiatek) [reply src/ui/page/maintenance.html]: it is a copy & paste from error.html page - I can amend

**Comment by m24murray** [comment]
Closing this as no longer needed.

**Pull request closed without merging**